scipy/spatial/distance.py

```diff
@@ -1153,7 +1153,8 @@ def pdist(X, metric='euclidean', p=2, w=None, V=None, VI=None):
     Y : ndarray
         Returns a condensed distance matrix Y.  For
         each :math:`i` and :math:`j` (where :math:`i<j<n`), the
-        metric ``dist(u=X[i], v=X[j])`` is computed and stored in entry ``ij``.
+        metric ``dist(u=X[i], v=X[j])`` is computed and stored 
+        in the :math:`ij` th entry.
```

Comment:
This isn't actually true, and that seemed to be the concern of #4472, although I am not sure that making it correct won't be more confusing than helpful. The distance between the `i`-th and `j`-th elements is actually stored, if I am not mistaken, in position:

```
i*n - i*(i+1)/2 + j - (i + 1)
```

To see where this comes from, for `i = 0` we store `n - 1` values, for `i = 1` that's `n - 2` values, and so on. So for a generic `i`, the first element will be stored after `i*n - i*(i+1)/2` elements, and since `j > i`, the first value stored for that `i` will be `j = i + 1`, hence the `+ j - (i + 1)`.

That formula should, by the way, be the same as the `{n \\ choose 2}...` of the `squareform` docstring.